stix_shifter/stix_translation/src/modules/aws_cloud_watch_logs/json/master_config.json

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
In the event that the from-STIX mappings change, would this file also need to be changed?

Reply:
yes